ui/app/AppLayouts/Communities/panels/PermissionsSettingsPanel.qml

```diff
@@ -49,7 +51,10 @@ StackView {
 
             text: qsTr("Add new permission")
 
-            onClicked: root.push(newPermissionView, StackView.Immediate)
+            onClicked:  {
+                console.log("COMMUNITY ID: " + communityDetails.id)
+                root.communityTokensStore.test()//root.push(newPermissionView, StackView.Immediate)
+            }
```

Comment:
some cleanup needed :)

Reply:
Yes! part of the commit to be removed! ;)